e2etests/web/text_editing/web/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE HTML>
```

Comment:
I think creating a full app for each test will result in too much boilerplate checked into the repo. I think a better approach is to have a couple of apps each containing a large number of integration tests. For example, we could have `e2etests/web/integration` for all integration tests that can share the same app configuration. If an e2e test needs specialized app configuration (e.g. PWA vs non-PWA packaging) we can create extra apps under `e2etests/web`, but I imagine there will be very few specialized cases like that.

Reply:
I have some concerns on that one. It will be more clear after everybody on the team fills out the sheet I am collecting about the integration tests.
From my experience in the other teams adding to much to this type of integration tests app will make the results flaky and hard to troubleshoot during an error. 
- We need a very simple app to see if flutter run -d web-server --local-engine load a page in the given os/browser.
- For every function we need to test the function only, for example a text editing test
shouldn't fail with other 15 tests if icons stop loading. Ideally one, in reality a handful should fail.
- If one wants to test things like navigation/history and everybody shared the same app, the other developers additions to the app can fail the previously written scenarios.
- If we need to navigate to a certain location in order to test every time app opens, the test can break not only it's functionality regress but also the steps in  between regress. For example a keyboard test will fail more often if you need to scroll down and select a page 'keyboard' sub app, before you come to keyboard tests.
- I propose to have at least 9-10 different apps we use. we can discuss more on the sheet.

Reply:
@yjbanov 
We had an offline discussion and here are the agreements we have:
- we want to reduce the number of files we have.
- felt will on the fly add the drive file. This way we will be able to remove the drive file. (I'll do it in the next PRs)
- e2e will have a notation of 'testMain' and 'main' so we can merge the application file (main.dart) and the e2e tests.
- we will still keep all the tests under e2etests since in the future we can make roller to run all the integration tests here and roll the PR from engine to flutter accordingly.
- I'll rename the package to a more general name so new web engine integration tests won't need to create their own package (their own pubspec and everything). We will only create a 
new package if we need a specific configuration. So we will have 2-3 of them at most.
- In the future PRs I'll make felt to run these tests automatically even if it is run under web_ui with felt test command.
- Felt test should give a proper log for the failure ideally with the location of the test so web
engine developers won't have to worry about it's location.